docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,95 @@
+# Running Flink-Job-Cluster in Kubernetes
+
+## Overview
+
+to run a Flink-Job-Cluster, you first have to deploy the Flink-Operator. For that, follow the [user_guide](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md)
+
+## Prerequisites
+
+This Chart requires you to have Flink Operator Running in your Cluster.
+It is advised to use at least the following image version of flink-operator: ```gcr.io/flink-operator/flink-operator:v1beta1-6```
+since it enables you to use annotations for the Flink-Job-Cluster components e.g. JobManager, TaskManager and Job.
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Prepare a Flink operator image. You should use at least ```gcr.io/flink-operator/flink-operator:v1beta1-6```.
+
+2. Deploy the Flink Operator Helm Chart, follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator)
+
+3. Prepare a Flink-Job-Cluster image. You should use ```flink:latest``` or ```flink:1.9.3``` which has been tested and confirmed to work with flink-job-cluster helm chart.
+
+3. Use the following command to install the flink-job-cluster chart:
+
+  ```bash
+  helm template --namespace [NAMESPACE] [RELEASE_NAME] [CHART] -f values.yaml |kubectl apply -f -
```

Comment:
IIUC, this `values.yaml` file is an example, so it it not part of the helm chart, it should be under the [examples/](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/examples) dir.

Mention that this is an example, and elaborate what needs to be included in the `values.yaml` file, i.e., the spec.

Reply:
Thank you for reviewing the guide! I have adjusted almost all of your suggestions however I am not entirely sure whether or not the values.yaml is just an example. It should be included in the helm-chart itself because it has the minimum configuration settings that are needed to succesfully deploy the Flink job cluster in a reliable state. As far as I know, values.yaml is always included in the helm-chart folders. Please correct me if I am wrong. We can also put an example at examples/dir but in my opinion it would be better to just include it in the chart.